src/Gaffer/SplinePlug.cpp

```diff
@@ -125,7 +125,7 @@ void monotoneCubicCVsToBezierCurve( const typename T::PointContainer &cvs, typen
 		typename T::YType nextSlope;
 
 
-		const typename T::Point *pNext;
+		const typename T::Point *pNext = nullptr;
```

Comment:
It's a pity MSVC can't see that we don't use `pNext` at all on the paths where it's uninitialised, because it means we're now masking warnings that better compilers would give us if we _did_ use it uninitialised in the future. Same goes for the previous commit. Not suggesting we do anything different, just a bit of gentle grumbling :)

Reply:
>  just a bit of gentle grumbling
Gentle grumbling heard, MSVC can sometimes be a bad influence ☹️